stdlib/2/UserList.pyi

```diff
@@ -1,9 +1,10 @@
-from typing import Iterable, MutableSequence, TypeVar, Union, overload
+from typing import Iterable, MutableSequence, TypeVar, Union, overload, Sequence
 
 _T = TypeVar("_T")
 _S = TypeVar("_S")
 
 class UserList(MutableSequence[_T]):
+    data: Sequence[_T]
```

Comment:
This is actually always a real `List`.

Reply:
I was following the example of UserDict, which sets its data type to `Mapping[...]`. Looking at the code, it is guaranteed to always be a Dict. Should I change that as well?

Reply:
That would be great!

Reply:
Done